packages/cli-repl/src/smoke-tests.ts

```diff
@@ -378,6 +379,22 @@ export async function runSmokeTests({
     }
   }
   if (wantPerformanceTesting) {
+    perfResults.push({
+      info: {
+        test_name: 'executable_size',
+        tags: [...commonPerfTags],
+      },
+      created_at: new Date().toISOString(),
+      completed_at: new Date().toISOString(),
```

Comment:
do we want a different completition value?

Reply:
I think this would mainly be relevant for tasks that actually measure durations